Makefile

```
@@ -99,6 +99,13 @@ valgrind:
             --log-file=/tmp/valgrind-output \
             python3 -m pytest --no-memcheck -vv --valgrind --valgrind-log=/tmp/valgrind-output
 
+.PHONY: valgrind-leak
+valgrind-leak:
```

```suggestion
valgrind-leak:
	python3 -c "import pytest_valgrind" > /dev/null 2>&1 || python3 -m pip install pytest-valgrind
```
Install pytest-valgrind if it is missing, like the 'valgrind' target does